src/main/java/org/opentripplanner/standalone/configure/ConstructApplicationFactory.java

```diff
@@ -87,6 +90,9 @@ public interface ConstructApplicationFactory {
 
   StreetLimitationParameters streetLimitationParameters();
 
+  @Nullable
+  LuceneIndex luceneIndex();
```

Comment:
I think this is creating the index - reindexing for every request?

Reply:
You should pass it in using the builder if you do not want it to be created every time.